src/System.Runtime.Numerics/tests/BigInteger/Comparison.cs

```diff
@@ -553,6 +554,11 @@ private static void VerifyComparison(BigInteger x, UInt64 y, int expectedResult)
                 Assert.Equal(x.GetHashCode(), ((BigInteger)y).GetHashCode());
                 Assert.Equal(x.ToString(), ((BigInteger)y).ToString());
             }
+            else
+            {
+                Assert.NotEqual(x.GetHashCode(), ((BigInteger)y).GetHashCode());
```

Comment:
If a change made a bunch of values fail this assertion, then it's likely enough that the hash code algorithm has been significantly weakened to be worth examining the possibility. As such while it's strictly inconclusive rather than a clear sign of a flaw, it may be worth keeping if it passes with the current values.